presentation/src/main/res/values-nl/strings.xml

```diff
@@ -203,7 +225,9 @@
     <string name="voucher_send_email_description">De voucher is verzonden per mail</string>
     <string name="voucher_qr_code_subtitle">Dit is de QR-code van de Voucher</string>
     <string name="voucher_qr_code_subtitle_with_fund_name">%s\nDit is uw  in de vorm van een QR-code</string>
-    <string name="voucher_qr_code_description">Laat de aanbieder deze QR-Code scannen om een betaling te kunnen doen</string>
+    <string name="voucher_qr_code_description">Dit is uw voucher in de vorm van een QR-code\nlaat de aanbieder deze QR-code scannen om een betaling te kunnen doen.</string>
```

Comment:
\nlaat should be '\nLaat'

A new line should always start with a capital letter

Reply:
Seems to not be fixed